code/player/Player.Jobs.cs

```diff
@@ -7,11 +9,140 @@ public partial class Player
     [BindComponent]
     public Jobs.PlayerJob Job { get; }
 
+    //The default cooldown time for setting the forgiveness timer
+    float cooldownTime => 30.0f;
+
+    /// <summary>
+    /// The cooldown after leaving a job to prevent being assigned to a new job too quickly
+    /// </summary>
+    public TimeUntil JobCooldown { get; set; }
```

Comment:
Naming any `TimeUntil` variables as `TimeUntilJobCooldown` helps readability. Cooldown is very ambiguous here, maybe `TimeUntilStartNextJob` would be more clear